api/v1alpha1/applicationset_types.go

```diff
@@ -75,6 +75,7 @@ type ClusterGenerator struct {
 	// Clusters today are stored as Kubernetes Secrets, thus the Secret labels will be used
 	// for matching the selector.
 	Selector metav1.LabelSelector `json:"selector,omitempty"`
+	Values   map[string]string    `json:"values,omitempty"`
```

Comment:
Only a minor request: could you add a comment for the `Values` line, something like:
```
// Values contains key/value pairs which are passed directly as parameters to the template 
```

Reply:
No problem.